src/app/routes/applications/governance-list/governance-list.component.scss

```diff
@@ -0,0 +1,4 @@
+.mat-menu-item[disabled], .mat-menu-item[disabled]::after,
+.mat-menu-item[disabled] .mat-icon-no-color, .mat-menu-item-submenu-trigger[disabled]::after  {
```

Comment:
Is it needed to add: `.mat-menu-item-submenu-trigger[disabled]::after` and `.mat-menu-item[disabled]::after` ? I don't see a case for it that's why i'm asking :)

Reply:
Yeah, somehow it needs to tag these classes to custom change the color of the text to disabled